ingest/ingest_pipeline.py

```diff
@@ -719,6 +746,17 @@ def main() -> None:
         arguments["cell_metadata_file"] = arguments["annotation_file"]
         # IngestPipeline initialization expects "name" and not "cluster_name"
         arguments["name"] = arguments["cluster_name"]
+
+
+    if "ingest_differential_expression" in arguments:
+        # DE may use metadata or cluster file for annots BUT
+        # IngestPipeline initialization assumes a "cell_metadata_file"
+        arguments["cell_metadata_file"] = arguments["annotation_file"]
```

Comment:
This is a vestige from `de.py` handling.  We should theoretically be able to remove `cell_metadata_file` / `annotation_file`, but that doesn't seem worth doing here.  Retaining it for now seems like reasonable tech debt.

Reply:
(This tech debt is now resolved.)